spec/features/user_attempts_to_view_invisible_profile_spec.rb

```diff
@@ -1,6 +1,8 @@
 require "rails_helper"
 
 RSpec.describe "user attempts to view invisible profile" do
```

Comment:
we can probably drop this whole spec, yeah?

Reply:
Done 👍 80b36f121be3839997da383cf93f451a79780127